ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.19__amd64-nightly.yaml

```diff
@@ -4384,6 +4384,76 @@ tests:
   steps:
     cluster_profile: nutanix-qe-dis
     workflow: agent-qe-nutanix-disconnected
+- as: aws-short-cert-rotation-f9
```

Comment:
AFAIK, if a test can run on both amd64 and arm64, it's better to run them on multi with mixure arch, to save the costs.   CC @jianlinliu

Reply:
The short cert rotation doesn't worker nodes architecture of cloud platform, so mixture nodes is not suitable, for a cloud platform, we only choose one architecture, either amd64 or arm64.

Reply:
Then suggest to use arm64, that would help save our cost.

Reply:
@jianlinliu I will test 2 jobs on mixed arch then I will update jobs accordingly